tests/e2e/Services/Sites/SitesBase.php

```diff
@@ -329,6 +329,35 @@ protected function getTemplate(string $templateId)
             'content-type' => 'application/json',
             'x-appwrite-project' => $this->getProject()['$id'],
         ]);
+
+        // Fetch latest commit from GitHub API if template has provider info
+        if (
```

Comment:
getTempalte functions is meant to be exact representation of endpoint. doing this affects the logic and changes response format.

Let's do this in a separate helper function, maybe `helperGetLatestCommit(string $owner, string $repository)`